OpenDreamRuntime/AtomManager.cs

```diff
@@ -248,6 +249,10 @@ public void SetAppearanceVar(IconAppearance appearance, string varName, DreamVal
                 value.TryGetValueAsString(out var name);
                 appearance.Name = name ?? string.Empty;
                 break;
+            case "desc":
+                value.TryGetValueAsString(out var desc);
+                appearance.Desc = desc ?? string.Empty;
```

Comment:
Setting `desc` to an invalid value like `123` sets it to `null` in BYOND. `Desc` needs to be a nullable string, `string?`.